daemon/src/model/cfd.rs

```diff
@@ -1664,6 +1726,13 @@ impl Dlc {
             .map(|output| Amount::from_sat(output.value))
             .unwrap_or_default()
     }
+
+    pub fn own_script_pubkey(&self, role: Role) -> Script {
```

Comment:
I don't think `own` is entirely correct here. If I am passing in the other party's role, it is not true. Perhaps this is better?

```suggestion
    pub fn script_pubkey_for(&self, role: Role) -> Script {
```